src/org/dotnet/ci/pipelines/scm/VSTSPipelineScm.groovy

```diff
@@ -60,9 +60,9 @@ class VSTSPipelineScm implements PipelineScm {
                                 // Sets up the project field to the non-parameterized version
                                 url(this.getGitUrl())
                                 // Set the refspec to be the parmeterized version
-                                refspec('${GitRefSpec}')
+                                refspec('${VSTSRefspec}')
                                 // Set URL to the parameterized version
-                                url('${GitRepoUrl}')
+                                url('${VSTSRepoUrl}')
```

Comment:
Still incorrect casing I think.

Reply:
VSTSRepoUrl matches the casing above:                 
stringParam('VSTSRepoUrl', this.getGitUrl(), 'VSTS repo to clone.')

Which should be necessary as GithubPipelineScm.groovy has similar one:
stringParam('GitRepoUrl', Utilities.calculateGitURL(this._project), 'Git repo to clone.')